specs/common/error-signalling.md

```diff
@@ -0,0 +1,155 @@
+# Error Signalling
+
+<!-- copied from ../serving/knative-api-specification-1.0.md#error-signalling -->
+
+Knative APIs use the
+[Kubernetes Conditions convention](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties)
+to communicate errors and problems to the user. Note that Knative customizes the
+general Kubernetes recommendation with a `severity` field, and does not include
+`lastTransitionTime` for scalability reasons. Each user-visible resource
+described in Resource Overview MUST have a `conditions` field in `status`, which
+MUST be a list of `Condition` objects of the following form. Fields in the
+condition which are not marked as REQUIRED may be omitted to indicate the
+default value (i.e. a Condition which does not include a `status` field is
+equivalent to a `status` of `"Unknown"`). The actual API object types in an
+OpenAPI document may be named `FooCondition` to allow better code generation and
```

Comment:
s/may/might/
I think this sentence also need something like "For example,..." to make it clear that we're just providing guidance and not something normative.